boards/arm/pinetime_devkit0/doc/index.rst

```diff
@@ -119,7 +119,7 @@ In order to get started with Zephyr on the PineTime, you can use the
 board-specific sample:
```

Comment:
Last minor change, but +1 otherwise. Thanks again for your time working on this!

 ```suggestion
basic button sample:
```